### ISSUE TITLE
no code block shown with step or list

### ISSUE THREAD

**Issue by clnperez** [derekparker/delve#495]
1. What version of Delve are you using (`dlv version`)?
> dlv version
Delve Debugger
Version: 0.11.0-alpha
Build: 26ce16f1db3fdd6fdd915f2ec7d11e4bfe72baed

2. What version of Go are you using? (`go version`)?
go version go1.6 linux/amd64

3. What operating system and processor architecture are you using?
Fedora 23, x86_64

4. What did you do?

terminal A: 
$ sudo dlv attach <pid>
$ break <path:file:linenum>
$ continue

terminal B:
$ [command that will hit the breakpoint]

terminal A:
[breakpoint hit and info shown: "<path:file:linenum> (hits total:1) (PC: 0x58a35f)"]
$ list

5. What did you expect to see?
in terminal A, I expected to see the source code and an arrow pointing to the current line at which execution had stopped

6. What did you see instead?
just the breakpoint info

I tried rebuilding from master since it looked like the 'list' command was added at the end of last year, but it still seems to be broken.

**Comment by clnperez** [comment]
Interestingly, I do see the arrow for a some assembly, so I'm not sure what I'm missing:

```
> syscall.Syscall6() /usr/local/go/src/syscall/asm_linux_amd64.s:53 (PC: 0x5fee1a)
    48:		MOVQ	a4+32(FP), R10
    49:		MOVQ	a5+40(FP), R8
    50:		MOVQ	a6+48(FP), R9
    51:		MOVQ	trap+0(FP), AX	// syscall entry
    52:		SYSCALL
=>  53:		CMPQ	AX, $0xfffffffffffff001
    54:		JLS	ok6
    55:		MOVQ	$-1, r1+56(FP)
    56:		MOVQ	$0, r2+64(FP)
    57:		NEGQ	AX
    58:		MOVQ	AX, err+72(FP)
```

**Comment by aarzilli** [comment]
I can't reproduce this. When you see the breakpoint notification:

```
> main.someFunction() /path/filename.go:30 (hits goroutine(1):1 total:1) (PC: 0x13ee8)
```

This should be followed by the listing with `=>` on the current instruction. Do you see it or not? Can you confirm that `/path/filename.go` exists? Whats the output of the `goroutines` command inside `dlv`?

**Comment by clnperez** [comment]
@aarzilli, thanks for the quick response. I didn't get a notification so sorry for the delay.

No, I don't see the listing with => on the current instruction. I don't see a listing at all. 

My harddrive died and I set this up again on the new one, and it's working. I have a theory that because I built the binary in a container and the source locations weren't the same, that that was my issue. I'm curious as to why it still successfully added the breakpoint, but, for now, I'm good. Thanks again!
